app/templates/components/forms/orders/order-form.hbs

```diff
@@ -279,12 +279,13 @@
         {{t 'Acknowledgement'}}
       </h4>
       <p>
-        {{t
-          'By clicking "Pay Now", I acknowledge that I have read and that I agree with all the terms of services and privacy policy of Open Event.'}}
+        {{t 'By clicking "{{orderState}}", I acknowledge that I have read and that I agree with the ' 
```

Comment:
Invalid syntax

Reply:
Thanks a lot for correcting so many mistakes :)
But it was working perfectly and no console errors either.